src/analysis/full-analysis.jl

```diff
@@ -26,13 +22,13 @@ function run_full_analysis!(server::Server, uri::URI; onsave::Bool=false, token:
             end
             id = hash(run_full_analysis!, hash(analysis_unit))
             if onsave
-                debounce(id, FULL_ANALYSIS_DEBOUNCE) do
-                    throttle(id, FULL_ANALYSIS_THROTTLE) do
+                debounce(id, get_config(server.state.config_manager, ["performance", "full_analysis", "debounce"])) do
```

Comment:
We should probably change the interface like `get_config(server, "performance", "full_analysis", "debounce")` letting inference unroll internal recursion while avoiding allocating arrays.

Reply:
I agree with you about the interface.

As for the debounce/throttle, this implementation was from original implementation.
I will change it if it wasn’t intentional.

https://github.com/aviatesk/JETLS.jl/blob/266faff1d6e4b7cca5c0641a5f9779f2e7cdc8f7/src/analysis/full-analysis.jl#L29-L30